discovery-infra/day2.py

```diff
@@ -215,3 +222,79 @@ def _day2_cluster_create_params(openshift_version, api_vip_dnsname):
         "api_vip_dnsname": api_vip_dnsname,
     }
     return params
+
+def get_ocp_cluster(args):
```

Comment:
lets keep day2.py for day2 on clud operations, and move your need code  to a new file ( for example ocp.py

Reply:
done